src/styles/block/cards.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  margin-bottom: 48px;
+
+  &__image {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    width: 166px;
+    height: 16px;
+  }
+
+  &__review {
+    color: #060b35;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 166px;
+    height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-amount {
+    color: #060b35;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__price-price {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__buy {
+    width: 166px;
+    height: 40px;
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    background-color: #00acdc;
+    border-radius: 5px;
+    color: #fff;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+  }
```

Comment:
I think stretching a container with a margin is a bad idea. It's better to stretch with padding on the container itself.